ci-operator/step-registry/ipi/conf/aws/route53/ipi-conf-aws-route53-commands.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+curl -L https://github.com/mikefarah/yq/releases/download/3.3.0/yq_linux_amd64 -o /tmp/yq && chmod +x /tmp/yq
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+
+
+# subnet, AZs and hosted zone
+allsubnetids="${SHARED_DIR}/allsubnetids"
+hostedzoneid="${SHARED_DIR}/hostedzoneid"
+availabilityzones="${SHARED_DIR}/availabilityzones"
+if [ ! -f "${allsubnetids}" ] || [ ! -f "${availabilityzones}" ] || [ ! -f "${hostedzoneid}" ]; then
+    echo "File ${allsubnetids} or ${availabilityzones} or ${hostedzoneid} does not exist."
+    exit 1
+fi
+
+echo -e "subnets: $(cat ${allsubnetids})"
+echo -e "AZs: $(cat ${availabilityzones})"
+echo -e "hosted zone: $(cat ${hostedzoneid})"
+
+CONFIG_ROUTE53_HOSTEDZONE="${SHARED_DIR}/install-config-route53.yaml.patch"
+cat > "${CONFIG_ROUTE53_HOSTEDZONE}" << EOF
+publish: External
+platform:
+  aws:
+    hostedZone: $(cat "${hostedzoneid}")
+    subnets: $(cat "${allsubnetids}")
+controlPlane:
+  platform:
+    aws:
+      zones: $(cat "${availabilityzones}")
+compute:
+- platform:
+    aws:
+      zones: $(cat "${availabilityzones}")
+EOF
+
+/tmp/yq m -x -i "${CONFIG}" "${CONFIG_ROUTE53_HOSTEDZONE}"
+
+# Print install-config:
+echo "install-config.yaml:"
+echo '{pullSecret: dummy, proxy: {httpProxy: dummy, httpsProxy: dummy}, additionalTrustBundle: dummy}' > /tmp/dummy
```

Comment:
1. Personally I think we do not have to print install-config anywhere.
2. Even we check the install-config.yaml files, this patch will add `proxy` and `additionalTrustBundle` there, especially when install-config.yaml never set these items, this will easy confuse our debugging.